src/server/index.test.jsx

```diff
@@ -171,3 +142,76 @@ describe('Server', () => {
     });
   });
 });
+
+describe('Server HTTP Headers', () => {
+  let statusRequest;
+
+  beforeAll(async () => {
+    statusRequest = await request(server).get('/status');
+  });
+
+  it(`should not contain 'x-powered-by'`, () => {
+    const headerKeys = Object.keys(statusRequest.headers);
+    expect(headerKeys).not.toContain('x-powered-by');
+  });
+
+  it(`should have 'x-frame-options' set to 'DENY'`, () => {
+    validateHttpHeader(statusRequest.headers, 'x-frame-options', 'DENY');
+  });
+
+  it(`should have X-DNS-Prefetch-Control set to 'off' `, () => {
+    validateHttpHeader(statusRequest.headers, 'x-dns-prefetch-control', 'off');
```

Comment:
Basically it is a header that tells the browser it can run DNS requests in parallel to page calls and prefetch dns queries to links on the page. This can gracefully improve performance at a security cost since the dns queries can leak. It is recommended to set this setting to off for security reasons unless you really need to aggressively reduce latency. We can always set this setting to on if it is necessary.

Reply:
> This header tells the browser it can run DNS requests in parallel to page calls and prefetch dns queries to links on the page. This can gracefully improve performance at a security cost since the dns queries can leak. It is recommended to set this setting to off for security reasons unless you really need to aggressively reduce latency.